### ISSUE TITLE
`ObjectDataSet::getData()` with instance and `Each` + `Nested` combination fails

### ISSUE THREAD

**Issue by arogachev** [yiisoft/validator#455]
Trying to call `$dataSet->getData()` here:

https://github.com/yiisoft/validator/blob/66a5c531c726d03395146921f664de745d3dcc67/tests/TestEnvironments/Php81/DataSet/ObjectDataSetTest.php#L192

fails with error:

```
Error : Typed property Yiisoft\Validator\Tests\TestEnvironments\Support\Data\Charts\Chart::$points must not be accessed before initialization
```

Currently only rules are tested In the mentioned file.  Need to test data as well.

**Comment by vjik** [comment]
It's OK, because property `$data` in `Chart` not initilaized.

**Comment by arogachev** [comment]
Why `$data` property is even needed here?

**Comment by vjik** [comment]
Sorry, `$points` in Chart not initilaized.

**Comment by arogachev** [comment]
Do you mean something like this? 

https://github.com/yiisoft/validator/blob/66a5c531c726d03395146921f664de745d3dcc67/tests/Support/Data/ObjectWithNestedObject.php#L20

**Comment by vjik** [comment]
Error because property `$point` in `Chart`  object is not initialized. It's OK behaviour.

**Comment by arogachev** [comment]
Will be fixed in #445.
